website/docs/essentials/testing/listen_provider.dart

```diff
@@ -10,13 +10,13 @@ final provider = Provider((_) => 'Hello world');
 void main() {
   test('Some description', () {
     final container = createContainer();
-    /* SNIPPET START */
+/* SNIPPET START */
```

Comment:
Could you revert those changes?
The macro relies on the indent to trim redundant whitespaces.

That's the difference between:

```dart
foo();
```

and
```dart
    foo();
```

Reply:
Actually let me do that right now


```suggestion
    /* SNIPPET START */
```